pages/input-controls/override-styles.md

```diff
@@ -1,36 +1,34 @@
 ---
 layout: default
-title: Override Styles
+title: Override default styles
 nav_order: 4
 parent: Input Controls
 has_children: false
 ---
 
-# Overriding default styles
+# Override default styles
 
-Currently, all input controls are shipped with default styles. 
-These default styles are applied to the input controls when they are rendered in the HTML element container.
-In order to prevent any conflicts with your app's styles, the input control styles have been scoped to the input
-control container by using a prefix class. The prefix class is `jv-`.
+All input controls are shipped with default styles. These default styles are applied to the input controls when they are rendered in the HTML element container. To prevent any conflicts with your application's styles, the input control styles have been scoped to the input control container only by using the prefix `jv-` on all CSS classnames.
 
 ## The default styles
 
 The package `@jaspersoft/jv-ui-components` provides a stylesheet that contains the default styles for the input controls.
-This file can be located in the following path: `@jaspersoft/jv-ui-components/dist/jv-ui.css`.
-By default, it is imported in the sample app's `index.css` and `App.tsx` files. E.g:
+This file can be located at the following path: `@jaspersoft/jv-ui-components/dist/jv-ui.css`.
+It is imported in the sample application's `index.css` and `App.tsx` files as follows:
 ```css
 @import "@jaspersoft/jv-ui-components/dist/jv-ui.css";
 ```
+Optionally, there is a configuration that enables you to use the default styles for the input controls components only. When this configuration is used, the other styles in your application will not be affected by the Visualize Components CSS:
```

Comment:
could we remove the word '`optionally`'?
this isn't optional, it is mandatory to apply the proper styling of our components.

Reply:
Done.